src/server/configs/common.ts

```diff
@@ -25,9 +25,13 @@ export default {
         extended: false,
     },
     expressTrustProxyNumber: 2,
-    workers: (process.env.WORKERS && parseInt(process.env.WORKERS)) || 1,
-    fetchingTimeout: 95 * 1000,
-    singleFetchingTimeout: 95 * 1000,
+    workers: process.env.WORKERS ? parseInt(process.env.WORKERS, 10) : 1,
+    fetchingTimeout: process.env.DATA_FETCHING_TIMEOUT_SEC
```

Comment:
Maybe it's better to use milliseconds in env variables too?

DATA_FETCHING_TIMEOUT_MS
DATA_SINGLE_FETCHING_TIMEOUT_MS

Reply:
Ok, fixed